src/test/java/edu/kis/vh/nursery/RyhmersJUnitTest.java

```diff
@@ -40,11 +41,16 @@ public void testIsFull() {
 		boolean result = ryhmer.isFull();
 		Assert.assertEquals(true, result);
 	}
+	/*Powstał podczas robienia punktu 4/5 *
+	Konstruktor domyślny DefaultCountingOutRyhmer() używa IntLinkedList, która w metodzie isFull zwraca false, poniewaz
+	lista nie ma zmiennej odpowiedzialnej za ilość liczb jaką przechowuje,zatem nigdy nie będzie pełna
+	zatem test nie przejdzie ze względu na linie 41/
+	 */
 
 	@Test
 	public void testPeekaboo() {
 		DefaultCountingOutRyhmer ryhmer = new DefaultCountingOutRyhmer();
-		final int EMPTY_STACK_VALUE = -1;
+		final int EMPTY_STACK_VALUE = 0;

```

Comment:
Original constant should be used (it is better to use a constant from stacks interface).